scripts/sanity_chk/arches/arm.ini

```diff
@@ -10,6 +10,7 @@ platforms = qemu_cortex_m3 frdm_k64f arduino_due nucleo_f103rb stm32_mini_a15
             cc3220sf_launchxl frdm_kl25z disco_l475_iot1 nucleo_l432kc
             nucleo_f413zh stm32l496g_disco stm32f4_disco 96b_carbon_nrf51
             stm32f469i_disco nucleo_f412zg sam4s_xplained
+            stm32f469i_disco olimex_stm32_e407
```

Comment:
Yeah something went wrong with a rebase there, it duplicates the last line with "stm32f469i_disco" and adds the "olimex_stm32_e407".

I'll push a fix in a minute.